File: 236A_BoyOrGirl.java
import java.util.*;

class Solution {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        Set<String> set = new HashSet<String>(Arrays.asList(sc.nextLine().split("")));
        if (set.size() % 2 == 0) {
            System.out.println("CHAT WITH HER!");
        } else {
            System.out.println("IGNORE HIM!");
        }
        sc.close();
    }
}

File: 800/cpp/236A_BoyOrGirl.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
	string s;
	cin >> s;
	set <char> distinct;
	for (int i = 0; i < s.length(); i++) {
		distinct.insert(s.at(i));
	}
	if (distinct.size() % 2 == 0) {
		cout << "CHAT WITH HER!\n";
	} else {
		cout << "IGNORE HIM!\n";
	}
}